features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Create/GraphQL/Steps/ProvideGraphQL.jsx

```diff
@@ -94,6 +94,13 @@ export default function ProvideGraphQL(props) {
             });
     }
 
+    useEffect(() => {
+        const { inputValue } = apiInputs;
+        if (inputValue) {
+            onDrop([inputValue]);
+        }
+    }, []);
```

Comment:
Since `apiInputs` coming from a prop , shall we add it to effect's dependencies array ?

please go through this [doc](https://reactjs.org/docs/hooks-effect.html)

Reply:
Changed in 394c7cf